README.md

```diff
@@ -1,6 +1,14 @@
 # simple-chat
 A simple chat app using Angular and socket.io
 
+# Deployment Guidelines - Frontend
+
+[simple-chat Deployment](https://github.com/mbrandau/simple-chat/blob/master/frontend/README.md) 
+
+# Contribution Guidelines 
+
+[Contribute](CONTRIBUTING.md) 
```

Comment:
Add a short sentence like `Please feel free to contribute to this project! Just follow out [contribution guide](CONTRIBUTING.md).`